### PR TITLE
Add encryption of identifiers

### PR DESCRIPTION
**Pull request by svenvanderburg**
Add encryption of identifiers.
The idea is that this will be used by an RPC function when needed. I did not implement the passing of the salt from configuration, but I guess that is trivial and can be done when needed.

### CODE REVIEW

File: v6_carrier_py/encryption.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+from typing import List
+
+import pandas as pd
+
+
+def fixed_length_string(string: str, desired_length: int = 128):
+    """
+    Either truncate or padd a string with 'a' so that it is of the specified
+    length.
+    TODO: This is token from Chang's repo. Do we really want to support this or
```
Comment (svenvanderburg) [inline v6_carrier_py/encryption.py]: @dsmits what do you think?

**Comment by svenvanderburg** [comment]
Coverage is decreasing. I would vote for ignoring it, definitely at this prototype stage. In general I'm a fan of testing for the sake of testing instead of testing for the sake of increasing coverage. But if you really want to I can add more tests.

File: v6_carrier_py/encryption.py (hunk 1 above)
Comment (dsmits) [reply v6_carrier_py/encryption.py]: Yeah this is one of those details from changs repo that I think isn't totally necessary to implement. Let's enforce a 128 length salt, and make sure they are randomly generated when we create them.

Comment (dsmits) [reply v6_carrier_py/encryption.py]: You don't really use it at the moment do you?

**Review by dsmits — APPROVED** [review]
Wat mij betreft mag dat aanvullen tot 128 bytes er wel uit. Verder prima hoor!

File: v6_carrier_py/encryption.py (hunk 1 above)
Comment (svenvanderburg) [reply v6_carrier_py/encryption.py]: Ok, I'll take it out :)

**Pull request merged**